Sources/eppo/ConfigurationRequester.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation;
+
+let RAC_CONFIG_URL = "/api/randomized_assignment/v3/config"
+
+enum ConfigurationRequesterError: Error, CustomNSError {
+    case invalidJSON(String)
+    case parsingError(String)
+
+    static var errorDomain: String { return "ConfigurationRequesterError" }
+    
+        var errorCode: Int {
+        switch self {
+        case .invalidJSON:
+            return 100 
+        case .parsingError:
+            return 101
+        }
+    }
+}
+
+class ConfigurationRequester {
+    private let httpClient: EppoHttpClient;
+
+    public init(httpClient: EppoHttpClient) {
+        self.httpClient = httpClient
+    }
+
+    public static func decodeRACConfig(from jsonString: String) throws -> RACConfig {
```

Comment:
Is this `public` for testing purposes? If so, I wonder if there is a better modifier. Perhaps [`Internal`](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol/) may be a good fit.

Reply:
nice! yea `internal` seems like a great part of the language